zebrad/tests/common/lightwalletd/send_transaction_test.rs

```diff
@@ -177,25 +177,25 @@ pub async fn run() -> Result<()> {
         assert_eq!(response, expected_response);
     }
 
-    // The timing of verification logs are unreliable, so we've disabled this check for now.
-    //
-    // TODO: when lightwalletd starts returning transactions again:
-    //       re-enable this check, find a better way to check, or delete this commented-out check
-    //
-    //tracing::info!("waiting for mempool to verify some transactions...");
-    //zebrad.expect_stdout_line_matches("sending mempool transaction broadcast")?;
+    // Give some time for transaction to be sent to mempool
+    std::thread::sleep(std::time::Duration::from_secs(30));
+
+    // Check if some transaction is sent to mempool
+    tracing::info!("waiting for mempool to verify some transactions...");
+    zebrad.expect_stdout_line_matches("sending mempool transaction broadcast")?;
```

Comment:
This code waits twice, once for 30 seconds, and then waits for the log. So I don't think the 30 seconds wait is needed.
```suggestion
    // Check if some transaction is sent to mempool
    tracing::info!("waiting for mempool to verify some transactions...");
    zebrad.expect_stdout_line_matches("sending mempool transaction broadcast")?;
```

Reply:
I will confirm this locally.

Reply:
You are right, this is not needed and the test just passed locally without it.

Reply:
Actually, after some more runs i am seeing this error sometimes:

```
The application panicked (crashed).
Message:  all transactions from future blocks failed to send to an isolated mempool
Location: zebrad/tests/common/lightwalletd/send_transaction_test.rs:209
```

That was not happening with the sleep but i am not sure if this a coincidence.